frontend/tests/unit/components/layout/Tooltip.spec.ts

```diff
@@ -69,6 +69,16 @@ describe('Tooltip.vue', () => {
     expect(wrapper.find('[data-test="tooltip"]').exists()).toBeFalsy();
   });
 
+  it('shows tooltip on touch', async () => {
+    const wrapper = await createWrapper();
```

Comment:
Can we rename this test case to something like "allows controlling tooltip visibility based on touch events" since it covers both touchstart & touchend events?
Same goes for the hover test case above.

Reply:
sure :heavy_check_mark: